src/main/java/lotto/domain/constant/LottoRank.java

```diff
@@ -4,23 +4,27 @@
 
 public enum LottoRank {
 
-    FIRST(6, 2_000_000_000),
-    SECOND(5, 1_500_000),
-    THIRD(4, 50_000),
-    FOURTH(3, 5_000),
-    MISS(0, 0);
+    FIRST(6,false, 2_000_000_000),
```

Comment:
공백을 한 칸 추가하면 좋을 것 같아요.
IntelliJ IDEA를 사용 중이시면 자동 포맷팅 단축키(`Ctrl(Cmd) + Alt + L`)를 활용해보시는 걸 추천 드립니다.
```suggestion
    FIRST(6, false, 2_000_000_000),
```